st2api/st2api/controllers/v1/keyvalue.py

```diff
@@ -61,19 +63,22 @@ def __init__(self):
         self.get_one_db_method = self._get_by_name
 
     @jsexpose(arg_types=[str, str, str, bool])
-    def get_one(self, name, scope=SYSTEM_SCOPE, user=None, decrypt=False):
+    def get_one(self, name, scope=None, user=None, decrypt=False):
         """
             List key by name.
 
             Handle:
                 GET /keys/key1
         """
-        self._validate_scope(scope=scope)
+        if not scope:
+            scope = SYSTEM_SCOPE
```

Comment:
> but I assume it has something to do with the way pecan handles argument so the actual default argument value is overridden with None when calling the method.

Yes, this is what I noticed. The behavior is inconsistent. Even if supply a value for the API call using a query param, pecan defaulted to SYSTEM_SCOPE. I didn't chase this but I decided to be on safe side.